alignak/macroresolver.py

```diff
@@ -138,7 +138,7 @@ def init(self, conf):
         # Try cache :)
         # self.cache = {}
 
-    def _get_macros(self, s):
+    def _get_macros(self, string):
```

Comment:
Same think, not name the variable 'string', but something like 'chain' or 'stringvalue'